buildall

```diff
@@ -13,19 +14,61 @@ GCR_BASENAME=gcr.io/bitnami-containers/minideb
 
 mkdir -p build
 
+import() {
+    SOURCE=$1
+    TIMESTAMP=$2
+    ./import "$SOURCE" "$TIMESTAMP"
+}
+
+log() {
+    echo "$@" >&2
+}
+
 for DIST in $DISTS; do
    [ -f debootstrap/$DIST ] || (echo "buildall: Unknown distribution: $DIST" && exit 1)
-   echo "============================================"
-   echo "Building $BASENAME:$DIST"
-   echo "============================================"
+   current_ts="$(date -u +%Y-%m-%dT%H:%M:%S.%NZ)"
+   if docker pull $BASENAME:$DIST > /dev/null; then
+       target_ts="$(docker inspect $BASENAME:$DIST | jq --raw-output ".[0].Created")"
+       target_id="$(docker inspect $BASENAME:$DIST | jq --raw-output ".[0].Id")"
+   else
+       target_ts="$current_ts"
+       target_id=
+   fi
+   log "============================================"
+   log "Building $BASENAME:$DIST"
+   log "============================================"
    ./mkimage build/$DIST.tar $DIST
-   IMPORTED=$(docker import --change "CMD /bin/bash" build/$DIST.tar)
-   echo "============================================"
-   echo "Running tests for $BASENAME:$DIST"
-   echo "============================================"
+   IMPORTED=$(import build/$DIST.tar "$target_ts")
```

Comment:
Naming suggestion: getImageIDWithCustomTimestamp or something that speaks by itself, I was super confused about the work `import` passing a tarball and a timestamp. Although using comments can help to understand the whole point behind `import` so it is not important to change it